src/ManagedServiceIdentity/test/Get-AzFederatedIdentityCredentials.Recording.json

```diff
@@ -0,0 +1,162 @@
+{
+  "Get-AzFederatedIdentityCredentials+[NoContext]+List+$GET+https://management.azure.com/subscriptions/0336439f-0e9d-44ec-975e-62accb9b3901/resourceGroups/fic-rg-yr1q7dts/providers/Microsoft.ManagedIdentity/userAssignedIdentities/fic-identity-6scr1l/federatedIdentityCredentials?api-version=2022-01-31-preview+1": {
+    "Request": {
+      "Method": "GET",
+      "RequestUri": "https://management.azure.com/subscriptions/0336439f-0e9d-44ec-975e-62accb9b3901/resourceGroups/fic-rg-yr1q7dts/providers/Microsoft.ManagedIdentity/userAssignedIdentities/fic-identity-6scr1l/federatedIdentityCredentials?api-version=2022-01-31-preview",
+      "Content": null,
+      "isContentBase64": false,
+      "Headers": {
+        "x-ms-unique-id": [ "16" ],
+        "x-ms-client-request-id": [ "99bc9aa8-392a-4d1e-bcea-d35c79f13bd4" ],
+        "CommandName": [ "Get-AzFederatedIdentityCredentials" ],
+        "FullCommandName": [ "Get-AzFederatedIdentityCredentials_List" ],
+        "ParameterSetName": [ "__AllParameterSets" ],
+        "User-Agent": [ "AzurePowershell/v0.0.0", "PSVersion/v7.1.3", "Az.ManagedServiceIdentity/0.1.0" ],
```

Comment:
have we increased user-agent version ("Az.ManagedServiceIdentity/0.1.0")?
If not, I think it would be something good to do.